Formula/gst-plugins-bad.rb

```diff
@@ -23,6 +23,8 @@ class GstPluginsBad < Formula
   depends_on "gettext"
   depends_on "gst-plugins-base"
   depends_on "openssl"
+  depends_on "libnice"
```

Comment:
As WebRTC support is relatively new in `gst-plugins-bad`, mentioned in this  [comment](https://github.com/Homebrew/homebrew-core/issues/25649#issuecomment-375729784), and may break often - shouldn't this be made an `:optional` dependency? Is it a similar situation with `srt`?

Reply:
@rakkesh As per Gstreamer documentation, "GStreamer Bad Plug-ins is a set of plug-ins that aren't up to par compared to the rest. They might be close to being good quality, but they're missing something - be it a good code review, some documentation, a set of tests, a real live maintainer, or some actual wide use". In short, anything in this package is already understood as being less stable than what is in the "good plug-ins" package and more stable than what is in the "ugly plug-ins" package. Therefore I don't think it is necessary to make WebRTC and SRT optional here, they have already been deemed good enough for the "bad plug-ins" package by the Gstreamer developers.

Reply:
Thanks for the explanation. My comment was not much of a review but just an enquiry. 
I had missed out on the more pressing [comment](https://github.com/Homebrew/homebrew-core/issues/25649#issuecomment-375714340) by @tschoonj, as he usually reviews `freedesktop` related formulae.

Reply:
@rakkesh Yes, I understand that. @jon-courtney already responded to that comment on the same thread. I'm just pointing out here that the "may break often" argument is redundant with the fact that these functionalities are provided in a package that already implies unstable functionalities. If we make WebRTC and SRT optional, then all other plugins in this package should also be made optional, and vice versa.

Reply:
The only other comment that I would add to what @sjmgarnier has said is that, since gst-plugins-bad is a collection of a large number of independent plugins, the individual user is probably less concerned about the stability of any individual component.  Odds are that s/he is relying on a small fraction of the elements provided by the package anyway.  What is more troubling (to me at least) is when the GStreamer project advertises the presence of a plugin but it can't be found via Homebrew.  So I personally favor the inclusion of all the plugins, all other things being equal.

I suppose an argument in favor of NOT including any individual plugin by default might be if its inclusion pulls in a particularly large dependency.  If that is a particular concern with the `webrtcbin` and `srt` plugins (although I don't know that it should be), then I suppose we could mark them as `:recommended` instead.